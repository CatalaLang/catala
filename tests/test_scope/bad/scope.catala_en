## Article

```catala
declaration scope A:
  context a content integer
  context b content integer
  context c content boolean

scope A:
  definition c equals false
  definition a under condition c consequence equals 42
  definition a under condition not c consequence equals 0
  definition b under condition not c consequence equals 1337
  definition b under condition not c consequence equals 0
```
