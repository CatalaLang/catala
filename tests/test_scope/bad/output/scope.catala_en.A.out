[ERROR] There is a conflict between multiple exceptions for assigning the same variable.
[ERROR] 
[ERROR] This justification is true:
[ERROR]   --> test_scope/bad/scope.catala_en
[ERROR]    | 
[ERROR] 14 |   definition b under condition not c consequence equals 0
[ERROR]    |                                ^^^^^
[ERROR]    + Article
[ERROR] 
[ERROR] This justification is true:
[ERROR]   --> test_scope/bad/scope.catala_en
[ERROR]    | 
[ERROR] 13 |   definition b under condition not c consequence equals 1337
[ERROR]    |                                ^^^^^
[ERROR]    + Article
